### ISSUE TITLE
Create temp folder in Stoat/a3e/

### ISSUE THREAD

**Issue by auermich93** [tingsu/Stoat#20]
The current setup misses to create a `temp` folder in `Stoat/a3e/`. If you don't create the folder manually, you end up in an endless loop saying 'waiting for config file'.

**Comment by tingsu** [comment]
Yes, you are right. Thank you for letting us know this issue. We will fix it.
